packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -430,9 +439,10 @@ class HeapTreeViewState extends State<HeapTree>
                 Column(
                   children: [
                     _buildAllocationsControls(themeData),
-                    Expanded(
+                    const SizedBox(height: denseRowSpacing),
```

Comment:
Shouldn't the analyzer/linter have mentioned that a const was needed?  Surprised didn't see that in latest.

Reply:
this is a const? When I don't use a const where I could be, I get an analyzer warning and a squiggley in intellij.

Reply:
I used VSCode.  Should be the same?

Reply:
Are you saying you got an analysis error for the code I wrote? what was the analysis error?